src/Models/Images/Images.php

```diff
@@ -40,11 +40,11 @@ public function all(RequestOpts $requestOpts = null): array
      * Returns all image objects.
      *
      * @see https://docs.hetzner.cloud/#resources-images-get
-     * @param string|null $name
+     * @param RequestOpts $requestOpts
      * @return APIResponse|null
      * @throws \LKDev\HetznerCloud\APIException
      */
-    public function list(RequestOpts $requestOpts = null): ?APIResponse
+    public function list(RequestOpts $requestOpts): ?APIResponse
```

Comment:
Why did you introduce this breaking change?

Reply:
Since you're doing `$requestOpts->buildQuery()` in the next line, `$requestOpts` cannot be `null` ;-)

Reply:
🤔  yep right, but then we should check it because otherwise, this would be a breaking change.

Reply:
BTW: I'm not sure if it's even possible to pass `null` when it's typehinted with `RequestOpts` (not `?RequestOpts`)...
Anyway, I cannot include a check since I don't know what to do in case of failure.

Reply:
@ThomasLandauer i changed it :)